doc/en/components/grids/_shared/export-excel.md

```diff
@@ -156,13 +156,13 @@ public webGridExportEventFreezeHeaders(args: CustomEvent<IgcExporterEvent>): voi
 <!-- React -->
 <!-- ComponentStart: Grid, TreeGrid, HierarchicalGrid -->
 ```tsx
-function exportEventFreezeHeaders(grid: IgrGridBaseDirective, args: IgrExporterEvent) {
+function exportEventFreezeHeaders(args: IgrExporterEventArgs) {
     args.detail.options.freezeHeaders = true;
-}
+  }
 
 <IgrGridToolbar key="toolbar">
```

Comment:
`key` property is no longer needed neither in the toolbar nor in its actions. Could you remove them all?

Reply:
removed